tox.ini

```diff
@@ -73,26 +77,37 @@ commands_pre =
   pip install -e {toxinidir}/ext/opentelemetry-ext-http-requests
   pip install -e {toxinidir}/ext/opentelemetry-ext-jaeger
   pip install -e {toxinidir}/ext/opentelemetry-ext-wsgi
+  pip install -e {toxinidir}/ext/opentelemetry-ext-flask[test]
   pip install -e {toxinidir}/examples/opentelemetry-example-app
 
 commands =
+
+  black --check --diff .
+
 ; Prefer putting everything in one pylint command to profit from duplication
 ; warnings.
-  black --check --diff .
-  pylint opentelemetry-api/src/opentelemetry \
-         opentelemetry-api/tests/ \
+  pylint \
+         opentelemetry-api/src/opentelemetry \
          opentelemetry-sdk/src/opentelemetry \
-         opentelemetry-sdk/tests/ \
-         ext/opentelemetry-ext-azure-monitor/examples/ \
-         ext/opentelemetry-ext-azure-monitor/src/ \
-         ext/opentelemetry-ext-azure-monitor/tests/ \
-         ext/opentelemetry-ext-http-requests/src/ \
-         ext/opentelemetry-ext-http-requests/tests/ \
+         ext/opentelemetry-ext-azure-monitor/examples \
+         ext/opentelemetry-ext-azure-monitor/src/opentelemetry \
+         ext/opentelemetry-ext-http-requests/src/opentelemetry \
          ext/opentelemetry-ext-jaeger/src/opentelemetry \
-         ext/opentelemetry-ext-jaeger/tests/ \
-         ext/opentelemetry-ext-wsgi/tests/ \
-         examples/opentelemetry-example-app/src/opentelemetry_example_app/ \
-         examples/opentelemetry-example-app/tests/
+         ext/opentelemetry-ext-wsgi/src/opentelemetry  \
+         ext/opentelemetry-ext-flask/src/opentelemetry  \
+         examples/opentelemetry-example-app/src/opentelemetry_example_app
+
+; The tests need extra pylint invocations, otherwise we get name clashes within the "tests" packages
```

Comment:
Any ideas on how we could structure this to avoid the name collision? ...and avoid `test-util`?

Reply:
Linking the relevant pylint issue: https://github.com/open-telemetry/opentelemetry-python/issues/210.

Reply:
The name collision issue was no longer present already since the move to the extra common folder, I just thought it cleaner. But I removed all structural toxfile changes now from the PR.